docker/checkout.bash

```diff
@@ -43,11 +43,11 @@ cd ${dir}/src
 if [[ "$BRANCH" = "develop" ]]; then
       git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-msgs.git --branch $BRANCH
       git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-utils.git --branch $BRANCH
-      git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-messenger.git --branch $BRANCH
+      git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-messenger.git --branch fix/map_msg_name	
```

Comment:
Reminder to clean this up before we merge this PR

Reply:
changed it back